src/DotVVM.Framework/Compilation/Binding/ExpressionBuildingVisitor.cs

```diff
@@ -105,6 +105,31 @@ protected override Expression VisitLiteralExpression(LiteralExpressionBindingPar
             return Expression.Constant(node.Value);
         }
 
+        protected override Expression VisitInterpolatedStringExpression(InterpolatedStringBindingParserNode node)
+        {
+            var target = new MethodGroupExpression() {
+                MethodName = nameof(String.Format),
+                Target = Registry.Resolve(nameof(String))
```

Comment:
```suggestion
                Target = typeof(string)
```

If you'd define your own string, we still want to use System.String (as C# does: https://sharplab.io/#v2:C4LglgNgNAJiDUAfAAgJgIwFgBQyDMABGgQMrABOYAdgOYEDeOBzRhy6ADAQLIAU1wAgA8AlAyYtJyAOwEAJACJ6QgL4KA3BOYqcKoA=)